tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/LiferayServerCore.java

```diff
@@ -77,6 +77,8 @@
 public class LiferayServerCore extends Plugin
 {
 
+    public static final String BUNDLE_COMPILE_MARKER_TYPE = "com.liferay.ide.server.core.BundleCompileProblemMarker";
```

Comment:
Because there could be other reasons why the bundle didn't build besides the COMPILE error